setup.py

```diff
@@ -9,7 +9,7 @@
 
 install_requires = [
     'cffi',
-    'chainer >=1.23, !=2.0.0a1, !=2.0.0b1',
```

Comment:
We still need `, !=2.0.0a1, !=2.0.0b1`, as these verions are incompatible with ChainerMN.